packages/main/src/i18n/messagebundle.properties

```diff
@@ -37,6 +37,8 @@ DATEPICKER_DATE_ACC_TEXT=Date
 #XACT: DatePicker 'Open Picker' icon title
 DATEPICKER_OPEN_ICON_TITLE=Open Picker
 
+FILE_UPLOADER_BROWSE_TEXT=Browse
```

Comment:
The text entry in openui5 is (it`s easy for the translation delivery if the key matches, also there are 3 dots after the browse):
FILEUPLOAD_BROWSE=Browse...